app/mailers/digest_mailer.rb

```diff
@@ -11,7 +11,7 @@ def digest_email
     # set sendgrid category in the header using smtp api
     # https://docs.sendgrid.com/for-developers/sending-email/building-an-x-smtpapi-header
     if ForemInstance.sendgrid_enabled?
-      smtpapi_header = { category: ["Digest Email"] }.to_json
+      smtpapi_header = { "category": "Digest Email" }.to_json # rubocop:disable Lint/SymbolConversion
```

Comment:
This isn't necessary because `.to_json` will stringify the whole thing.
```ruby
[1] pry(main)> { category: "Digest Email" }.to_json
=> "{\"category\":\"Digest Email\"}"
[2] pry(main)> puts _
{"category":"Digest Email"}
=> nil
```


```suggestion
      smtpapi_header = { category: "Digest Email" }.to_json
```